images/hook/entrypoint.sh

```diff
@@ -116,8 +61,15 @@ spec:
                 values:
                 - $NODE_NAME
 EOF
-    kubectl --namespace=monitoring patch deployment influxdb -p "$(cat $TMPFILE)"
-    rm $TMPFILE
+        kubectl --namespace=monitoring patch deployment influxdb -p "$(cat $TMPFILE)"
```

Comment:
I might be missing something, but if the intent of this patch is to schedule influxdb on a particular node, then hasn't it already been scheduled before this runs? I.e. "rig upsert" above created the pod, it got scheduled, then this patch runs but basically has no effect because of "preferredDuringSchedulingIgnoredDuringExecution" affinity.

Reply:
You are right, we need to delete pod to reschedule it. Good catch!